benchmark-artifact/cpp-benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+ #####################################
+ # process_cloud.cpp
+ #####################################
+ add_executable(benchmark_process_cloud process_cloud.cpp)
+ target_compile_features(benchmark_process_cloud PRIVATE cxx_std_17)
+ target_link_libraries(benchmark_process_cloud PRIVATE ${PROJECT_NAME})
+
+ target_compile_options(benchmark_process_cloud PRIVATE -std=c++14 -Ofast)
```

Comment:
Why you declare C++17 with target_compile_features, but then pass C++14 with target_compile_options? This is why tests are failing, filesystem has become a member of std since C++17.

Reply:
ops..